assets/css/layout/_header.scss

```diff
@@ -0,0 +1,149 @@
+// Header styles
+
+header {
+    .brand {
+        a {
+            color: $black;
+            font-size: rem(48);
+            font-weight: $font-weight-normal;
+            padding-right: rem(16);
+        }
+
+        svg {
+            margin: rem(-8) rem(16) 0 rem(16);
+            width: rem(48);
+        }
+    }
+
+    .site-nav {
+        align-items: center;
+        background-color: $navy-light;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        padding: 0 rem(16);
+
+        // Toggle menu button on mobile design
+        // Override the default button styles from _base.scss and apply the new style
+        button {
+            background: none;
+            border: none;
+            border-radius: 0;
+            box-shadow: none;
+            color: $white;
+            font-size: rem(16);
+            padding: rem(8);
+        }
+
+        button::before {
+            content: none;
+        }
+        // End of overriding the default button styles from _base.scss
+
+        button:hover {
+            background-color: $orange-focus;
```

Comment:
I think the button needs a focus style as well — and the text colour should change on hover/focus (I'd use the same colour as the hover/focus states for the deskop nav links).

Reply:
@cindyli The text colour for the focus state is still white, it should be darker.

<img width="451" alt="Screen Shot 2020-03-25 at 11 12 06 AM" src="https://user-images.githubusercontent.com/605361/77565189-7a522480-6e89-11ea-9ff2-252c99e6380e.png">